packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -491,21 +503,74 @@ To edit platform code in an IDE see https://flutter.dev/developing-packages/#edi
 void _printPluginUpdatePubspecMessage(String pluginPath, String platformsString) {
   globals.printStatus('''
 You need to update $pluginPath/pubspec.yaml to support $platformsString.
-
 ''', emphasis: true, color: TerminalColor.red);
 }
 
 void _printNoPluginMessage() {
     globals.printError('''
 You've created a plugin project that doesn't yet support any platforms.
-
 ''');
 }
 
-void _printAddPlatformMessage(String pluginPath) {
+void _printPluginAddPlatformMessage(String pluginPath) {
   globals.printStatus('''
 To add platforms, run `flutter create -t plugin --platforms <platforms> .` under $pluginPath.
 For more information, see https://flutter.dev/go/plugin-platforms.
 
 ''');
 }
+
+/// returns a list of platforms which user has requested a platform and if it's not enabled
+List<String> _getPlatformWarningList(List<String> requestedPlatforms) {
+  final List<String> _disabledPlatform = <String>[
+  if (requestedPlatforms.contains('android') && !featureFlags.isAndroidEnabled)
```

Comment:
I think from the `create` command's perspective, android and iOS are always enabled. See https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/create.dart#L225-L239
So to match the logic, we probably shouldn't check here.

Reply:
@pedromassangocode asked to add them. I didn't noticed that so removing them makes sense to me as well.